jbake-core/src/main/java/org/jbake/app/FileUtil.java

```diff
@@ -41,6 +41,21 @@ public boolean accept(File pathname) {
         };
     }
 
+    /**
+     * Filters files based on their file extension - only find data files (i.e. files with .yaml or .yml extension)
+     *
+     * @return Object for filtering files
+     */
+    public static FileFilter getDataFileFilter() {
+        return new FileFilter() {
+
+            @Override
+            public boolean accept(File pathname) {
+                return "yaml".equalsIgnoreCase(fileExt(pathname)) || "yml".equalsIgnoreCase(fileExt(pathname));
```

Comment:
This does seem a little short sighted, sure yaml is a great start, but shouldn't jbake accept any of the other kinds of configuration options for data files? (json, properties, etc.)

Reply:
Yeah it is, however I felt this was an acceptable short cut at the time. This is specifically adding support for YAML data files, when others are added a more robust solution can be implemented.

Reply:
Fair enough